src/Arcus.Security.Core/Arcus.Security.Core.csproj

```diff
@@ -17,15 +17,19 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
We can do this bc it wasn't yet released.

Reply:
Do we have to do this? Since it's the same package shouldn't we rely on NuGet to handle this?

Reply:
This isn't about the Observability change, but about those other packages that were fixed on a v3.1.8 version; we changed that in observability and thought it will be a good thing to use that same approach here; splitting between netstandard and netcoreapp.

Reply:
I'll leave it up to @fgheysels to decide

Reply:
Dependencies, the pita in library development :)

- targetting a higher version of a dependent library means that consumers of Arcus must also use at least the version that is used by Arcus
- Does 3.1.8 not work on .NET Standard applications that do not target .NET Core 3.1 ?

Reply:
I think it's mostly bc we support(ed) .NET Core 2.2 throughout Arcus (and still do).

Reply:
If it's for .NET Core 2.2; ignore that since that's end-of-life

Reply:
Ok, I change it back then to 3.1.8 packages.